spaceship/lib/spaceship/du/utilities.rb

```diff
@@ -65,8 +65,9 @@ def video_resolution(video_path)
       video_infos = output.split("\n").select { |l| l =~ /Stream.*Video/ }
       raise "Unable to find Stream Video information from ffmpeg output of #{command}" if video_infos.count == 0
       video_info = video_infos[0]
-      res = video_info.match(/.* ([0-9]+)x([0-9]+),.*/)
-      raise "Unable to parse resolution information from #{video_info}" if res.count < 3
+      res = video_info.match(/.* ([0-9]+)x([0-9]+) .*/)
```

Comment:
this change is interesting. Do we have a test for it ? Is your output different from mine ? Do we need to support both ?